openarena/Program.cs

```diff
@@ -13,14 +13,18 @@ static void Main(string[] args)
         {
             Console.WriteLine("OpenArena for Azure PlayFab Multiplayer Servers");
             
+            // GSDK event handlers - we're setting them on the startup of the app
```

Comment:
Even thought this class only has 6 lines, when you add more detailed comments for the following lines, this method may become longer and lose the focus of those comments. If possible, I'd like to have two functions, one for GSDK related calls, and one for gameProcess calls. Then we can document them separately. For GSDK related calls, it's easier to summarize and show the customer that it only takes less than 5 lines to integrate with GSDK. For gameProcess related calls, it's easier to explain the creative way that you have for integration. 
The purpose is all about document and make sure the customer can capture the key content they are looking for before they leave the page. Our current GSDK sample don't have the structure and document because they are an afterthought to make it a public sample. Their initial idea was actually to validate GSDK and serve as testcases. We can reiterate on them later following this project's good practice.